metadata-ingestion/source_docs/athena.md

```diff
@@ -0,0 +1,62 @@
+# AWS Athena
+
+To install this plugin, run `pip install 'acryl-datahub[athena]'`.
+
+This plugin extracts the following:
+
+- List of databases and tables
+- Column types associated with each table
+
+```yml
+source:
```

Comment:
This is a bit overwhelming. Can we extract the parameters into a table or list below and say what each does? Configuration schema docs IMO should be separate from example